cc_plugin_glider/util.py

```diff
@@ -73,6 +73,9 @@ def _check_variable_attrs(dataset, var_name, required_attributes=None):
 
     # Get the expected attrs to check
     check_attrs = required_attributes or required_var_attrs.get(var_name, {})
+    if not options is None:
```

Comment:
Just a suggestion but I think this reads better. 

```suggestion
    if options is not None:
```